spec/lib/vanagon/engine/ec2_spec.rb

```diff
@@ -1,25 +1,33 @@
-require 'vanagon/engine/ec2'
-require 'vanagon/platform'
+begin
+  require 'aws-sdk'
+rescue LoadError
```

Comment:
Bundler groups are inclusionary, not exclusionary. With no flags passed, Bundler will install all  gems and groups:

```